src/lib/mongoBackend/MongoGlobal.cpp

```diff
@@ -1282,6 +1283,10 @@ bool entitiesQuery
       {
         exErr = invalidPolygon;
       }
+      else if (strncmp(exErr.c_str(), sortError, strlen(sortError)) == 0)
+      {
+        exErr = "Sort operation used more than the maximum RAM. You should create an index. Check Database Administration section at Orion documentation.";
```

Comment:
ok, so:

Check Database Administration section at Orion documentation.  =>
Check the Database Administration section in Orion documentation

Not perfect, but good enough ...